sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ not include preview features).
 
 ### Bugs Fixed
 
+* Fix the issue with logging Agent message, when the message has "in progress" status.
```

Comment:
Please add a link to the relevant GitHub issue (see examples of how we did it in previous releases)